modules/database/test/std/link/lnkCalcTest.c

```diff
@@ -157,7 +157,7 @@ static void testCalc()
 
 MAIN(lnkCalcTest)
 {
-    testPlan(0);
+    testPlan(30);
```

Comment:
oops.  This is an unrelated bug fix.

Reply:
@mdavidsaver Should I take any action on this?  Leave it as 30?  Revert back to 0?

Reply:
Is this is the only change in that commit? If so I will apply it directly to the 7.0 branch.

Reply:
> Is this is the only change in that commit? If so I will apply it directly to the 7.0 branch.

There were a few other changes, see 14140acd787e990f34bf1d4d827d6bb4903f282a

Reply:
Yeah, 14140acd787e990f34bf1d4d827d6bb4903f282a also added building of the link-type tests on RTEMS to the modules/database/test/std/link/Makefile. I can cherry-pick this whole commit into 7.0 (in which case @brendanchandler can drop it when he next rebases), or just change the `testPlan()` call there. @mdavidsaver your call.

Reply:
14140ac looks reasonable.

Reply:
@mdavidsaver I'm sure you're busy but your terse communication style doesn't let us know whether you think 14140acd787e990 should be cherry-picked into the 7.0 branch or left as a commit in @brendanchandler's rtems5 PR. The change you highlighted here was just part of that commit.

Reply:
My apologies Andrew Johnson, I will endeavor to express myself in this particular instance will what I hope you will find to be adequate verbosity.  Having looked though the commit in question (14140ac), I have formed the opinion that it is reasonable to apply in its entirety, and without delay.  Building and running the link type unittests on RTEMS targets is something which should already be happening.  The fact that this is not the case may be seen as an omission on my part, which I am grateful to Brendan Chandler for noticing and correcting.

Reply:
Cherry-picked into 7.0 as bbb4d86f787b74579062aa77db9a98aee8e7ba80